server/sonar-server/src/main/java/org/sonar/server/issue/index/IssueIndex.java

```diff
@@ -114,7 +114,7 @@
     SEVERITIES,
     STATUSES,
     RESOLUTIONS,
-    ACTION_PLANS,
+    DEPRECATED_ACTION_PLANS,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "DEPRECATED_ACTION_PLANS"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)